content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,7 +14,21 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+ Before upgrading to Kubernetes 1.24 , please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).
```

Comment:
Not sure why there are weird spaces.
```suggestion
Before upgrading to Kubernetes 1.24, please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).
```

And please address [previous suggestion](https://github.com/kubernetes/website/pull/34764#pullrequestreview-1033704600) to this line 4 weeks ago.